translator-src/nu/validator/htmlparser/cpptranslate/CppVisitor.java

```diff
@@ -1364,6 +1383,10 @@ public void visit(MethodCallExpr n, LocalSymbolTable arg) {
                 printer.print(cppTypes.stateLoopPolicies()[stateLoopCallCount]);
                 printer.print(">");
                 stateLoopCallCount++;
+            } else if (cppTypes.requiresTemplateParameter(n.getName())
+                    && "Tokenizer".equals(javaClassName)
```

Comment:
In my mind this would be easier to read if one would first check class name and then effectively the method name. But looks like you're just following the existing style, so fine.